desc/objectives/utils.py

```diff
@@ -153,14 +166,16 @@ def factorize_linear_constraints(constraints, objective_args):
         else:
             unfixed_args.append(arg)
             A_ = obj.derivatives["jac"][arg](jnp.zeros(obj.dimensions[arg]))
-            b_ = obj.target
+            # using obj.compute instead of obj.target to allow for correct scale/weight
+            b_ = -obj.compute(jnp.zeros(obj.dimensions[arg]))
```

Comment:
why is there a negative sign here?

Reply:
The residual is computed as f(x) = Ax-b, so f(0) = -b, so we need the negative sign to get the correct b